app/views/layouts/themes/default.html.haml

```diff
@@ -5,53 +5,58 @@
     %meta(name="viewport" content="width=device-width, initial-scale=1.0")
     = stylesheet_link_tag current_website.theme, media: 'all'
     = javascript_pack_tag "application"
+    %link{rel: "stylesheet", href: "https://fonts.googleapis.com/css?family=Rubik"}
     %script{src: "https://cdn.tailwindcss.com" }
   %body
-    %header#main-header
-      %div#header-logo-container
-        %a#menu-toggle{ href: "#"}
-        .website-event-logo
-        %h1
-          = link_to(current_website.event.name, landing_path(website_event_slug))
-      %nav#main-nav
-        - current_website.pages.published.each do |page|
-          = link_to page.name, page_path(website_event_slug, page)
-        = link_to "Schedule", schedule_path(current_website.event)
-        = link_to "Program", program_path(website_event_slug)
-        = link_to "Sponsors", sponsors_path(website_event_slug)
+    - unless @page&.hide_header?
+      %header#main-header
+        %div#header-logo-container
+          %a#menu-toggle{ href: "#"}
+          .website-event-logo
+          %h1
+            = link_to(current_website.event.name, landing_path(website_event_slug))
```

Comment:
With the new decorator, I think this can be just current_website.name. There are a few other places this could be cleaned up in this view.